packages/contracts/contracts/maci/Tally.sol

```diff
@@ -88,24 +85,44 @@ contract Tally is TallyBase, IPayoutStrategy, Pausable {
     Mode pollMode
   ) payable TallyBase(verifierContract, vkRegistryContract, pollContract, mpContract, tallyOwner, pollMode) {}
 
-  /// @notice A modifier that causes the function to revert if the cooldown period is not over
-  modifier afterCooldown() {
+  /// @notice A modifier that causes the function to revert if the tallying is not over
+  modifier afterTallying() {
+    if (!isTallied() || tallyBatchNum == 0) {
+      revert VotesNotTallied();
+    }
+
+    _;
+  }
+
+  /// @notice A modifier that causes the function to revert if not within deposit window
+  modifier duringDepositWindow() {
+    if (!isTallied() || tallyBatchNum == 0) {
```

Comment:
Nitpick: can be use a modifier inside a modifier and remove a bit of code:

```solidity
modifier duringDepositWindow() afterTallying {
  // ...
}
```

or 

```solidity
modifier duringDepositWindow() {
   afterTallying(); 
  // ...
}
```